installer/alpine/build.sh

```diff
@@ -31,15 +33,15 @@ mv /etc/mkinitfs/features.d/virtio.modules.tmp /etc/mkinitfs/features.d/virtio.m
 
 # Make initramfs with features we think are spiffy
 # shellcheck disable=SC2016
-echo 'features="base ext2 ext3 ext4 keymap network packetrepo squashfs virtio eclypsium"' >/etc/mkinitfs/mkinitfs.conf
+echo 'features="base ext2 ext3 ext4 keymap network packetrepo squashfs virtio"' >/etc/mkinitfs/mkinitfs.conf
 kver=$(basename /lib/modules/*)
 mkinitfs -l "$kver"
-mkinitfs -o /assets/initramfs-vanilla "$kver"
+mkinitfs -o /assets/initramfs-"$arch" "$kver"
```

Comment:
this is an odd way to quote a variable. isn't it nicer to the reader to use `"/assets/initramfs-$arch"` instead?

Reply:
yes